app/apollo/type-defs.ts

```diff
@@ -1,5 +1,31 @@
 // Describe our GraphQL schema with type definitions
 export const typeDefs = `
+  type Movie {
```

Comment:
I borrowed this example type straight from https://www.npmjs.com/package/@neo4j/graphql just to make sure I had everything configured properly.